### PR TITLE
Write E2E test for Gitea and Grafana

### PR DESCRIPTION
**Pull request by RothAndrew**
## What/Why
* Write E2E test that ensures Gitea and Grafana come up cleanly
* Fix a lint error that pre-commit was picking up

### CODE REVIEW

**Comment by RothAndrew** [comment]
Ready for review & merge

File: cli/internal/packager/create.go [hunk 1]
```diff
@@ -75,7 +75,7 @@ func addLocalAssets(tempPath componentPaths, assets config.ZarfComponent) {
 	if len(assets.Charts) > 0 {
 		logrus.Info("Loading static helm charts")
 		_ = utils.CreateDirectory(tempPath.charts, 0700)
-		re := regexp.MustCompile("\\.git$")
+		re := regexp.MustCompile(`\.git$`)
```
Comment (YrrepNoj) [inline cli/internal/packager/create.go:78]: Was this breaking before? I almost expect to see some other changes to how `.git` files are being written when looking at this change.

Comment (RothAndrew) [reply cli/internal/packager/create.go:78]: `pre-commit` picked it up as a lint error. It worked fine before, and should still work fine, just without throwing a lint error.

**Comment by YrrepNoj** [comment]
Is there a particular reason why we're commenting out old sections of the e2e.sh script instead of just remove sections that have been successfully migrated?

**Comment by RothAndrew** [comment]
> Is there a particular reason why we're commenting out old sections of the e2e.sh script instead of just remove sections that have been successfully migrated? Not totally against it, just curious if we're expecting to do something with it later.

I could remove them entirely if you feel strongly about it. Since the whole file is destined to be deleted I didn't see it as a big deal, and having the commented out code helps me keep track of what we already did and what we still have to do.

**Comment by YrrepNoj** [comment]
> 

I don't feel that strongly about it. I would argue it's easier to see what we still need to do if we remove it and just look at what's left but at the end of the day, you're right that the file is getting deleted and it doesn't matter.

**Review by YrrepNoj — APPROVED** [review]

**Pull request merged**